README.md

```
@@ -204,6 +204,48 @@ $ gup export --output > gup.json
 $ gup import --file=gup.json
 ```
 
+### Migrate binaries to a new $GOBIN
+
+```shell
+$ gup migrate BEFORE_PATH AFTER_PATH [BINARY...]
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)